src/ui/party-ui-handler.ts

```diff
@@ -153,6 +159,7 @@ export default class PartyUiHandler extends MessageUiHandler {
   private partySlotsContainer: Phaser.GameObjects.Container;
   private partySlots: PartySlot[];
   private partyCancelButton: PartyCancelButton;
+  private PartyDiscardModeButton: PartyDiscardModeButton;
```

Comment:
```suggestion
  private partyDiscardModeButton: PartyDiscardModeButton;
```

Rename to camelCase pls thx